model/system/metrics.yaml

```diff
@@ -32,6 +32,22 @@ groups:
     unit: "{cpu}"
     attributes: []
 
+  - id: metric.system.cpu.utilization
+    type: metric
+    metric_name: system.cpu.utilization
+    stability: development
+    brief: Fraction of total CPU time spent in active modes, aggregated across all system CPU cores.
+    note: >
+      This metric provides a standardized and opinionated definition of CPU utilization.
+      It is defined as the proportion of total system CPU time spent in non-idle and non-iowait modes — that is, the time spent in user, system, nice, interrupt, and steal modes.
+      It is computed by aggregating `cpu.time` across all logical CPUs for these modes over a measurement interval, and dividing the result by the elapsed time and the `system.cpu.logical.count` to produce a normalized,
+      system-wide utilization value between 0.0 and 1.0: sum(rate(cpu.time{mode != "idle" and mode != "iowait"})) / system.cpu.logical.count
+      This represents time spent actively executing work, including time delayed by the hypervisor (steal), which reflects CPU demand in virtualized environments.
+      As a result, in cases of high CPU contention — such as 100% steal time — this metric will report 100% utilization, making such issues easy to detect.
```

Comment:
https://github.com/open-telemetry/semantic-conventions/pull/2123 is not merged yet but maybe we can put a note here about resource/entity association already (similar to https://github.com/open-telemetry/semantic-conventions/blob/main/model/k8s/metrics.yaml#L153-L154)

Reply:
Added https://github.com/open-telemetry/semantic-conventions/pull/2088/commits/b7ef1550900d53b5ae619b7cb6ac44f6f4cb2817